### PR TITLE
mysql8

### PR DESCRIPTION
**Pull request by odinuv**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,9 +8,9 @@ services:
 before_install:
   - docker-compose -v
   - docker-compose build
-  - docker-compose run --rm app /code/vendor/bin/phpcs --standard=psr2 -n --ignore=vendor --extensions=php .
-  - docker-compose run --rm app /code/vendor/bin/phpstan analyse --level=7 ./src ./tests
-  - docker-compose run --rm app
+  - docker-compose run app
+  - docker-compose rm -s -f
```
Comment (pivnicek) [inline .travis.yml]: Nice, I didn't know about this:)

File: src/Keboola/DbExtractor/Extractor/MySQL.php [hunk 2]
```diff
@@ -100,6 +100,10 @@ public function createConnection(array $params): PDO
         }
         $pdo->setAttribute(PDO::MYSQL_ATTR_USE_BUFFERED_QUERY, false);
         $pdo->exec("SET NAMES utf8;");
+        $infile = $pdo->query("SHOW VARIABLES LIKE 'local_infile';")->fetch(\PDO::FETCH_ASSOC);
```
Comment (pivnicek) [inline src/Keboola/DbExtractor/Extractor/MySQL.php]: I'm not sure I understand why we need `local_infile` to be `ON` for the extractor.
It doesn't use `LOAD DATA LOCAL`, is this maybe something that only the writer needs?

Comment (odinuv) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: correct, we don't, it's just needed for the tests and i got confused

Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/MySQL.php]: Ah yeah, sorry I forgot to mention about the tests, looks great!

**Comment by pivnicek** [comment]
I'll release this with https://github.com/keboola/db-extractor-mysql/pull/54

**Comment by odinuv** [comment]
yep, there are no changes, so there is no urgency

**Pull request merged**